gigadb/app/tools/files-metadata-console/tests/bats/compare_files.bats

```diff
@@ -0,0 +1,47 @@
+@test "Show usage when no arguments passed" {
+	run scripts/compare_files.sh
+	[ "$output" = "Usage: scripts/compare_files.sh <file_list> <search_directory>" ]
+  [ "$status" -eq 1 ]
+}
+
+@test "Non existing file list" {
+	run scripts/compare_files.sh tests/_data/compare_files/gaigh tests/_data/compare_files/user0
+	[ "$output" = "File list tests/_data/compare_files/gaigh not found!" ]
+  [ "$status" -eq 1 ]
+}
+
+@test "Non existing search directory" {
+	run scripts/compare_files.sh tests/_data/compare_files/user0files fasfh
+	[ "$output" = "Search directory fasfh not found!" ]
+  [ "$status" -eq 1 ]
+}
+
+@test "Empty file list" {
+	run scripts/compare_files.sh tests/_data/compare_files/emptylist tests/_data/compare_files/user0
+	[[ "$output" =~ "Files found from the list: 0" ]]
+	[[ "$output" =~ "Files not found from the list: 0" ]]
+  [ "$status" -eq 0 ]
+}
+
+@test "non empty and existing file list and search directory passed to the command" {
+	run scripts/compare_files.sh tests/_data/compare_files/user0files tests/_data/compare_files/user0
+	[[ "$output" =~ "Files found from the list: 6" ]]
+	[[ "$output" =~ "Files not found from the list: 1" ]]
+	[[ "$output" =~ "Files in tests/_data/compare_files/user0 not listed in tests/_data/compare_files/user0files: 2" ]]
+  [ "$status" -eq 0 ]
+}
+
+@test "when the file list has blank lines in it" {
+	run scripts/compare_files.sh tests/_data/compare_files/emptylines tests/_data/compare_files/user0
+	[[ "$output" =~ "Files found from the list: 6" ]]
+	[[ "$output" =~ "Files not found from the list: 1" ]]
+	[[ "$output" =~ "Files in tests/_data/compare_files/user0 not listed in tests/_data/compare_files/emptylines: 2" ]]
+  [ "$status" -eq 0 ]
+}
```

Comment:
When I executed the script:
```
% ./scripts/compare_files.sh tests/_data/compare_files/emptylines tests/_data/compare_files/user0
Files found for README_000000.txt:
tests/_data/compare_files/user0/README_000000.txt
Files found for lorem.ipsum:
tests/_data/compare_files/user0/lorem.ipsum
No files found for .env
Files found for a/b/some.file.txt:
tests/_data/compare_files/user0/a/b/some.file.txt
Files found for a/b/c/f1:
tests/_data/compare_files/user0/a/b/c/f1
Files found for a/b/c/f2:
tests/_data/compare_files/user0/a/b/c/f2
Files found for a/b/c/f3:
tests/_data/compare_files/user0/a/b/c/f3

Files in tests/_data/compare_files/user0 not listed in tests/_data/compare_files/emptylines:
singlefiledir/alone.txt not listed

Summary:
Files found from the list: 6
Files not found from the list: 1
Files in tests/_data/compare_files/user0 not listed in tests/_data/compare_files/emptylines: 1
```

**suggestion:** There `emptylines` file contains 7 files to searched in the dir `tests/_data/compare_files/user0`, the only file not in the `tests/_data/compare_files/user0 ` is  just `.env`, so in the assertion `[[ "$output" =~ "Files in tests/_data/compare_files/user0 not listed in tests/_data/compare_files/emptylines: 2" ]]`, the number should be `1`.